task_01.rst

```diff
@@ -0,0 +1,131 @@
+
+#############################################
+  Why I Love Python
+#############################################
+
+Why I love **Python**, a language worth knowing.
+
+
+
+
+##################
+Running It
+##################
+
+Python syntax is really quite simple to execute. Anyone can do it just by executing the
+``python`` command on the command line and invoking the interpreter.
+
+You can also execute python programs by calling the program filename such as:
+
+.. code:: console  
+
+   me@mymachine $ python my_file_name.py
+    
+
+
+##################
+The Syntax
+##################
+
+Python syntax is very easy to write. Take a look at this simple statement:
+
+.. code:: python
+
+   >>> print 'Hello!' 
+
+
+##################
+The People
+##################
+
+It's amazing how nice people are in the Python development world. Just as with any
+community, however, it's important to follow certain rules of decorum.
+
+- Be courteous to others
+
+- Always read the manual before asking questions
+
+- Be succinct in your questions
+
+  - Don't give unnecessary details if they aren't needed
+
```

Comment:
As you suggested, this extra line is unnecessary but that's OK. RST is going to format the document according to how it feels it should do so best.